_posts/2025-08-27-offer-multiple-ways-to-contact-you.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "I have a question, but I don't have WhatsApp!"
+permalink: /blog/2025/offer-multiple-ways-of-contact/
+author: Rian Rietveld
+categories: contact, multiple ways
+parent: Blog
+excerpt: "It’s better to offer multiple ways to get in touch, so people can choose what suits them. Think about your visitors first, not just what’s easiest for you.
+"
+date: 2025-08-27
+---
+You want to reach out to the customer service, and they say  *“Send us a WhatsApp message.”* What if you don't use WhatsApp, or cannot make a phone call, or use the chatbox? If you offer multiple contact options, you're leaving no people (customers) behind.
+{: .fs-6 .fw-300 }
+
+## Don’t make getting in touch their problem
+
+Let’s say someone wants to ask a quick question through your website, but the only way to do it is by sending a WhatsApp message.
+
+Sure, it’s convenient for you. One place to manage it all, and you reply when it suits you.
+
+Other examples:
+
+- You only list the phone number of your helpdesk.
+- You only offer a chat function, with a bot you hope can handle basic questions. And if not, it points people to the “FAQs”.
+- You require people to come in person to renew their passport.
+- You provide a clunky, hard-to-use form to submit a complaint, just because that’s how it’s set up in your customer information software.
+
+In all these cases, you’re forcing your way of working on the people trying to reach you. And it’s worth remembering: some of them can’t even use these options.
+
+So, what are the ways to communicate? And what are the pros and cons?
+
+## Different ways to get in touch
+
+### Social media and instant messaging
+
+Services like WhatsApp or X (formerly Twitter) can be useful, it’s widely used, low-barrier, and great for quick questions.
+
+But don’t assume everyone uses it. Many of people are moving away from WhatsApp due to Meta’s privacy issues. Some prefer Signal or don’t want to share personal stuff over WhatsApp at all.
```

Comment:
"But don't assume everyone uses [them]. Many people are moving away from social media services due to privacy issues. Some prefer more private services, like Signal, or don't want to share personal information over your chosen service."

- This has a lot of changes; but I think it's good to make this broader. It's not *just* Meta's privacy issues; there are privacy issues in many of these services.